tests/test_diaPipe.py

```diff
@@ -292,6 +293,18 @@ def check_diaObjects(bbox, wcs, diaObjects):
         # Verify that no objects inside the bounding box were removed
         self.assertEqual(nIn1, nIn0)
 
+    def test_make_empty_catalog(self):
+        """Check that an empty catalog has the correct format.
+        """
+        tableName = "DiaObject"
+        config = self._makeDefaultConfig(config_file=self.config_file.name)
+        diaPipeTask = DiaPipelineTask(config=config)
+        emptyDiaObjects = make_empty_catalog(diaPipeTask.schema, tableName=tableName)
+        emptyDf = pd.DataFrame(columns=["diaObjectId",])
+        emptyDf.set_index("diaObjectId")
+        convertedEmptyDiaObjects = convertTableToSdmSchema(diaPipeTask.schema, emptyDf, tableName=tableName)
+        self.assertSetEqual(set(convertedEmptyDiaObjects.columns), set(emptyDiaObjects.columns))
```

Comment:
If you're comparing two `set`, then `assertSetEqual` is redundant.
```suggestion
        self.assertEqual(set(convertedEmptyDiaObjects.columns), set(emptyDiaObjects.columns))
```